infrastructure/index.ts

```diff
@@ -70,6 +70,12 @@ const uploadsBucket = new aws.s3.Bucket("uploads-bucket", {
     website: {
         indexDocument: "index.html",
     },
+    corsRules: [{
+        allowedMethods: [
+            "GET",
+        ],
+        allowedOrigins: ["*"],
```

Comment:
not sure if we think `*` is too generous. it does need to be that though if we want the previews to work.